src/Packetery/Module/ShippingMethod.php

```diff
@@ -79,6 +79,13 @@ public function calculate_shipping( $package = [] ): void {
 	 * @return array<string|int, mixed>
 	 */
 	public function get_instance_form_fields(): array {
-		return [];
+		return [
+			'title' => [
+				'title'   => __( 'Method title', 'packeta' ),
```

Comment:
Objeví se to někde? Nechal bych tam původní `Packeta` nebo bych dal třeba `Packeta shipping method`

Reply:
Pokud máme title , který jde do překladové funkce, je potřeba zjistit,  kde se to uživateli zobrazí a podle toho zvolit adekvátní název .  Pokud se nikde nezobrazuje, proč je potřeba to mít v kódu ?